.changeset/short-feet-sing.md

```diff
@@ -0,0 +1,6 @@
+---
+"@comet/brevo-admin": patch
+"@comet/brevo-api": patch
+---
+
+Remove the field `Total contacts blocked` in the Target Groups query, because it is not delivered in the list request in Brevo anymore.
```

Comment:
```suggestion
Remove the `totalContactsBlocked` field from the `TargetGroup` type, because it is not delivered in the list request in Brevo anymore.
```